### PR TITLE
Translation to Spanish of the Quick Select algorithm

### PR DESCRIPTION
**Pull request by david-xander**
Well, I just translate this new algorithm into spanish also. I've found also an implementation in Python. I don't know if it's not a good idea to add that into this translation. I will add an Issue with some mispelled words and not so good translations of word that prone to confusion that I've found out in order to know if I can fix them all. Or if I have to make a PR for each.

Thank you.

### CODE REVIEW

File: es/Algoritmos de selección/Selección Rápida.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```
+arr[] = {8,2,11,7,9,1,3}
+Indexes: 0 1 2 3 4 5 6
+
+Digamos que k = 4. Y tenemos que encontrar el cuarto elemento menor.
+
+1. Escogiendo un pivote de forma aleatoria, tenemos 7
+2. Intercambiamos 7 con el último elemento y aplicamos el algoritmo de particionado.
+3. Después de aplicar la partición, todos los elementos menores que 7 se colocan a la izquierda y los mayores a la derecha.
+4. Como la posición de '7' es la cuarta, simplemente devolveremos la valor 7.
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:40]:
```suggestion
4. Como la posición de '7' es la cuarta, simplemente devolveremos el valor 7.
```

File: es/Algoritmos de selección/Selección Rápida.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Selección rápida o Quick Select
+
+### Definición del problema
+
+Given an array, find the kth largest / smallest element in linear time complexity.
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:5]: This is not being translated to Spanish. Please translate it.

**Review by Panquesito7 — COMMENTED** [review]
Great work! 😄👍

File: es/Algoritmos de selección/Selección Rápida.md (hunk 2 above)
Comment (david-xander) [reply es/Algoritmos de selección/Selección Rápida.md]: Oops!!! My mistake.
I've just fixed it. Thank you... I'll be more careful next time

File: es/Algoritmos de selección/Selección Rápida.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Selección rápida o Quick Select
+
+### Definición del problema
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:3]:
```suggestion
## Definición del problema
```

File: es/Algoritmos de selección/Selección Rápida.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Dada una matriz, encontrar el k-ésimo elemento mayor y menor con un una complejidad lineal de Tiempo
+
+### Enfoque
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:7]:
```suggestion
## Enfoque
```

File: es/Algoritmos de selección/Selección Rápida.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+- Selecciona un elemento que sirva de pivote aleatoriamente.
+- Aplica particiones como las utilizadas en el algoritmo de ordenamiento rápido o quick sort.
+- Después de realizar las particiones, el pivote se deberá colocar en su posición ordenada. Todos los elementos menores que el pivote quedarán a su izquierda y los mayores a su derecha.
+- Si el índice del pivote ordenado es k, entonces el pivote está en su k-ésimo elemento y devolveremos el número.
+- De otro modo, verifica si 'k' es mayor o menor y selecciona un nuevo pivote en el rango.
+- Repite los pasos hasta conseguir el k-ésimo elemento en la k-ésima posición.
+
+### Complejidad de Tiempo
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:16]:
```suggestion
## Complejidad de Tiempo
```

File: es/Algoritmos de selección/Selección Rápida.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+- `O(n^2)` En el peor de los casos
+
+- `O(n)` En el mejor de los casos
+
+- `O(n)` Rendimiento promedio
+
+### Nombre de su fundador
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:24]:
```suggestion
## Nombre de su fundador
```

File: es/Algoritmos de selección/Selección Rápida.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+- Este algoritmo fue desarrollado por Tony Hoare. También es conocido como el 'Algoritmo de Selección de Hoare'.
+
+### Ejemplo
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:28]:
```suggestion
## Ejemplo
```

File: es/Algoritmos de selección/Selección Rápida.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+Digamos que k = 4 y tenemos que encontrar el cuarto elemento menor:
+
+1. Escogiendo un pivote de forma aleatoria, obtenemos 7.
+2. Intercambiamos 7 con el último elemento y aplicamos el algoritmo de particionado.
+3. Después de aplicar la partición, todos los elementos menores que 7 se colocan a la izquierda y los mayores, a la derecha.
+4. Como la posición de '7' es la cuarta, simplemente devolveremos el valor 7.
+
+```
+
+### Enlaces de implementaciones de código
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:43]:
```suggestion
## Enlaces de implementaciones de código 
```

File: es/Algoritmos de selección/Selección Rápida.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+4. Como la posición de '7' es la cuarta, simplemente devolveremos el valor 7.
+
+```
+
+### Enlaces de implementaciones de código 
+
+- [C++](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/selecting/quickSelect.cpp)
+- [Python](https://programmerclick.com/article/92711720579/)
+
+### Enlaces de explicación en vídeo
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:48]:
```suggestion
## Enlaces de explicación en vídeo
```

File: es/Algoritmos de selección/Selección Rápida.md (hunk 3 above)
Comment (david-xander) [reply es/Algoritmos de selección/Selección Rápida.md:3]: I agree with you here 100%. There are some other suggestions that you're pointing out but that were already there in the original file merges in english. Maybe that should be fixed also. Let me know what do I have to do in that case.

File: es/Algoritmos de selección/Selección Rápida.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+Matriz[] = {8,2,11,7,9,1,3}
+Índices: 0 1 2 3 4 5 6
+
+Digamos que k = 4 y tenemos que encontrar el cuarto elemento menor:
+
+1. Escogiendo un pivote de forma aleatoria, obtenemos 7.
+2. Intercambiamos 7 con el último elemento y aplicamos el algoritmo de particionado.
+3. Después de aplicar la partición, todos los elementos menores que 7 se colocan a la izquierda y los mayores, a la derecha.
+4. Como la posición de '7' es la cuarta, simplemente devolveremos el valor 7.
+
```
Comment (Panquesito7) [inline es/Algoritmos de selección/Selección Rápida.md:40]:
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
Great work, @david-xander 🚀 Thank you for your contribution! 😄👍

**Pull request merged**